cmd/influx/main.go

```diff
@@ -27,6 +27,8 @@ var influxCmd = &cobra.Command{
 	Run:   influxF,
 }
 
+var maxTCPConnections = 3000
```

Comment:
this is a high number of concurrent requests, is this number based on benchmarks/metrics somewhere?

Reply:
i will make it 128 for now, see how it goes.